src/crun.h

```diff
@@ -30,6 +30,7 @@ struct crun_global_arguments
   bool debug;
   bool option_systemd_cgroup;
   bool option_force_no_cgroup;
+  bool is_krun;
```

Comment:
could we make it more generic (e.g. have `const char *handler`)?

Reply:
yes this would be better.

Reply:
Resolved in latest commit.